typescript/src/lib/util/functions/messages.ts

```diff
@@ -163,3 +163,12 @@ export async function promptConfirmation(message: Message, options: string | Pro
 	const response = (await message.send(options)) as Message;
 	return canReact(response) ? promptConfirmationReaction(message, response, options) : promptConfirmationMessage(message, response, options);
 }
+
+export async function promptForMessage(message: Message, sendOptions: string | MessageOptions, time = Time.Minute): Promise<string | null> {
+	// The cast is for mismatching overloads:
+	const response = (await message.channel.send(sendOptions as any)) as Message;
```

Comment:
```suggestion
	// TODO: v13 | Remove typecast
	const response = (await message.channel.send(sendOptions as any)) as Message;
```

So we can find it back easily with [ToDo Tree](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree)